Library/ViewModels/RootViewModel.swift

```diff
@@ -159,21 +159,27 @@ public final class RootViewModel: RootViewModelType, RootViewModelInputs, RootVi
       self.userSessionEndedProperty.signal,
       self.currentUserUpdatedProperty.signal
     )
-    .map { AppEnvironment.current.currentUser }
+    .map { _ in AppEnvironment.current.currentUser }
 
     let userState: Signal<(isLoggedIn: Bool, isMember: Bool), Never> = currentUser
       .map { ($0 != nil, ($0?.stats.memberProjectsCount ?? 0) > 0) }
       .skipRepeats(==)
 
-    let standardViewControllers = self.viewDidLoadProperty.signal.map { generateStandardViewControllers() }
-    let personalizedViewControllers = userState.map { generatePersonalizedViewControllers(userState: $0) }
+    let standardViewControllers = self.viewDidLoadProperty.signal.map { _ -> [RootViewControllerData] in
+      generateStandardViewControllers()
+    }
+    let personalizedViewControllers = userState.map { userState -> [RootViewControllerData] in
+      generatePersonalizedViewControllers(userState: (userState.isMember, userState.isLoggedIn))
```

Comment:
This was necessary to work around something that was deprecated in Swift called tuple shuffling: https://noahgilmore.com/blog/tuple-shuffling/